WebRoot/js/zimbraAdmin/statistics/view/ZaGlobalAdvancedStatsPage.js

```diff
@@ -79,19 +79,21 @@ function (){
     controller._helpButtonText = ZaMsg.helpCheckStatistics;
 };
 
+ZaGlobalAdvancedStatsPage.getDataTipText = function (item) {
+    var text = AjxMessageFormat.format(
```

Comment:
you can return directly instead of using a variable

Reply:
Done. Thanks.